1-org/envs/shared/org_policy.tf

```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
   parent = "organizations/${var.org_id}"
   title  = "default policy"
 }
+
+
+module "restrict_procotol_fowarding" {
```

Comment:
it is necessary to set `allow_list_length` to 1

- https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/variables.tf#L80-L83
- https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/list_constraints.tf#L165
- https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/list_constraints.tf#L181